app/app.js

```diff
@@ -103,7 +103,10 @@ if (module.hot) {
 
 // Chunked polyfill for browsers without Intl support
 if (!window.Intl) {
-  System.import('intl').then(() => render(translationMessages));
+  Promise.all([
+    System.import('intl'),
+    System.import('intl/locale-data/jsonp/en.js'),
```

Comment:
@mxstbr please review.
I don't see what should be added in the docs, i think it's okay to leave it like that.